File: net/Azure.Storage.Blobs.PerfStress/Core/CountOptions.cs
using Azure.Test.PerfStress;
using CommandLine;

namespace Azure.Storage.Blobs.PerfStress.Core
{
    public class CountOptions : PerfStressOptions
    {
        [Option('c', "count", Default = 500, HelpText = "Number of blobs")]
        public int Count { get; set; }
    }
}

File: common/azure-core-test-perf/src/main/java/com/azure/core/test/perf/CountOptions.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.azure.core.test.perf;

import com.beust.jcommander.Parameter;

/**
 * Represents the count option for performance tests.
 */
public class CountOptions extends PerfStressOptions {
    @Parameter(names = { "-c", "--count" }, description = "Number of items")
    private int count = 10;

    /**
     * Get the configured count for performance test.
     * @return The count.
     */
    public int getCount() {
        return count;
    }
}
